app/views/internal/broadcasts/show.html.erb

```diff
@@ -0,0 +1,39 @@
+<div class="row my-3">
+  <div class="col">
+    <div class="list-group-flush w-100">
+    <a class="list-group-item flex-column align-items-start">
+      <h3 class="mb-2">Title: <%= @broadcast.title %></h3>
+      <h3>Broadcast Type: <%= @broadcast.type_of %></h3>
+      <h3>Content:</h3>
+      <p>
+        <%= @broadcast.processed_html %>
+      </p>
+      <h3>Last Active On: <%= @broadcast.active_status_updated_at&.strftime("%b %d, %Y %H:%M UTC") %></h3>
+      <h3>Status:</h3>
+      <h3>
+      <span class="badge badge-<%= @broadcast.active? ? "success" : "warning" %>">
+        <%= @broadcast.active? ? "Active" : "Inactive" %>
+      </span>
+      <span class="sr-only">Broadcast Status</span>
+      </h3>
+    </a>
+    <hr>
+      <% if @broadcast.processed_html %>
+        <div>
+          </p><strong>Preview</strong></p>
```

Comment:
super tiny detail. This should be an open tag.
```suggestion
          <p><strong>Preview</strong></p>
```

Reply:
Great catch @maestromac!